templates/user/auth/oidc_wellknown.tmpl

```diff
@@ -0,0 +1,9 @@
+{
+	"issuer": "{{AppUrl | JSEscape | Safe}}",
+	"authorization_endpoint": "{{AppUrl | JSEscape | Safe}}login/oauth/authorize",
+	"token_endpoint": "{{AppUrl | JSEscape | Safe}}login/oauth/access_token",
+	"userinfo_endpoint": "{{AppUrl | JSEscape | Safe}}login/oauth/userinfo",
```

Comment:
Is `AppUrl` guaranteed to contain a trailing slash?

Reply:
yes, look at modules/setting/setting.go Where AppURL is `AppURL = strings.TrimRight(AppURL, "/") + "/"` (note the different capitalization, this var is passed to the template using the expected capitalization I've used in the template)